modules/articles/client/config/articles.client.routes.js

```diff
@@ -12,25 +16,56 @@ angular.module('articles').config(['$stateProvider',
       })
       .state('articles.list', {
         url: '',
-        templateUrl: 'modules/articles/client/views/list-articles.client.view.html'
+        templateUrl: 'modules/articles/client/views/list-articles.client.view.html',
+        controller: 'ArticlesListController',
+        controllerAs: 'vm'
       })
       .state('articles.create', {
         url: '/create',
-        templateUrl: 'modules/articles/client/views/create-article.client.view.html',
+        templateUrl: 'modules/articles/client/views/form-article.client.view.html',
+        controller: 'ArticlesController',
+        controllerAs: 'vm',
+        resolve: {
+          articleResolve: newArticle
```

Comment:
Yes, but it's different and it depends on the state, and that isn't always crystal clear and people may confuse that.